fv3_cap.F90

```diff
@@ -979,102 +827,41 @@ subroutine ModelAdvance_phase1(gcomp, rc)
 
     ! local variables
     type(ESMF_Clock)            :: clock
-    type(ESMF_TimeInterval)     :: timeStep
-    type(ESMF_Time)             :: startTime, stopTime, currTime
-
     integer                     :: urc
     logical                     :: fcstpe
     character(len=*),parameter  :: subname='(fv3_cap:ModelAdvance_phase1)'
     character(240)              :: msgString
 
-    integer                     :: date(6)
-
 !-----------------------------------------------------------------------------
 
     rc = ESMF_SUCCESS
 
     if(profile_memory) call ESMF_VMLogMemInfo("Entering FV3 ModelAdvance_phase1: ")
 
-    fcstpe = .false.
-    if( mype < num_pes_fcst ) fcstpe = .true.
-
-    ! Expecting to be called by NUOPC run method exactly once for every coupling
-    ! step.
-    ! Also expecting the coupling step to be identical to the timeStep for
-    ! clock_fv3.
-
-    call ESMF_ClockPrint(clock_fv3, options="currTime", &
-                         preString="------>Advancing FV3 phase1 from: ", unit=msgString, rc=rc)
-    if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
-
-    call ESMF_LogWrite(msgString, ESMF_LOGMSG_INFO, rc=rc)
-    if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
-
-!-----------------------------------------------------------------------
-!***  Use the internal Clock set by NUOPC layer for FV3 but update stopTime
-!-----------------------------------------------------------------------
-
-    ! Component internal Clock gets updated per NUOPC rules
     call ESMF_GridCompGet(gcomp, clock=clock, rc=rc)
     if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
 
-    ! The stopTime will be updated to be the next external coupling time
-    call ESMF_ClockGet(clock, currTime=currTime, stopTime=stopTime, rc=rc)
-    if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
-
-    ! Set the FV3-OCN coupling time to be stopTime in Clock that FV3 core uses
-    !call ESMF_ClockSet(clock_fv3, currTime=currTime, stopTime=stopTime, rc=rc)
-    call ESMF_ClockSet(clock_fv3,                    stopTime=stopTime, rc=rc)
-    if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
-
-    call ESMF_ClockPrint(clock_fv3, options="currTime", &
+    call ESMF_ClockPrint(clock, options="currTime", &
                          preString="entering FV3_ADVANCE phase1 with clock_fv3 current: ", &
```

Comment:
maybe change clock_fv3 to fv3 clock to avoid confusion as clock_fv3 is not used?

Reply:
Changed in https://github.com/NOAA-EMC/fv3atm/pull/450/commits/070bc74bd95043a11a78e678d06d83bae087dedf